src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -113,21 +111,22 @@ def get_top_events(user_query, params, event_limit, referrer):
             )
 
         def generate_top_transaction_query(events):
-            top_transaction_names = [
-                re.sub(r'"', '\\"', event.get("transaction")) for event in events
+            pairs = [
+                (event["project_id"], re.sub(r'"', '\\"', event["transaction"])) for event in events
             ]
-            top_transaction_as_str = ", ".join(
-                f'"{transaction}"' for transaction in top_transaction_names
-            )
-            return f"transaction:[{top_transaction_as_str}]"
+            conditions = [
+                f"(project_id:{project_id} transaction:{transaction})"
```

Comment:
I think this existed because transaction names could have spaces, which breaks up the `transaction:value` assignment. Should we add back quotes around the transaction name in line 118?

Reply:
ahh good catch 👍

Reply:
updated the test to check this case as well